rubygem-foreman_templates/rubygem-foreman_templates.spec

```diff
@@ -35,6 +35,10 @@ Requires: %{?scl_prefix}rubygem(git)
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildRequires: %{?scl_prefix_ruby}rubygems
+BuildRequires: %{?scl_prefix}rubygem(diffy)
+BuildRequires: %{?scl_prefix}rubygem(git)
```

Comment:
`git` also appears to be a build time dependency, as it's a hard dependency of this gem.

    Git::GitExecuteError: git version   2>&1:sh: git: command not found
    /usr/share/gems/gems/git-1.2.5/lib/git/lib.rb:700:in `command'
    /usr/share/gems/gems/git-1.2.5/lib/git/lib.rb:649:in `current_command_version'
    /usr/share/gems/gems/git-1.2.5/lib/git/lib.rb:659:in `meets_required_version?'
    /usr/share/gems/gems/git-1.2.5/lib/git.rb:30:in `<top (required)>'